scripts/src/linting/typeimports.ts

```diff
@@ -6,10 +6,13 @@ function isImportSpecifier(spec: es.ImportDeclaration['specifiers'][number]): sp
 }
 
 function specToString(spec: es.ImportSpecifier) {
-  if (spec.imported.name === spec.local.name) {
-    return spec.imported.name;
+  if (spec.imported.type === 'Identifier')  {
+    if (spec.imported.name === spec.local.name) {
+      return spec.imported.name;
+    }
+    return `${spec.imported.name} as ${spec.local.name}`;
   }
-  return `${spec.imported.name} as ${spec.local.name}`;
+  return ''
```

Comment:
I'm not too familiar with this repo, but is it safe to return an empty string?

Reply:
Updated version: https://github.com/source-academy/modules/pull/314/files#diff-bb324eb5ce667eb484caea68fc23fc5bbdd519260c678545d4624c258a52b3f5R15

Reply:
I am not familiar with the scripts used, but should be fine right? CC @leeyi45.

In any case, we'll merge this first and fix the regressions (if any) later, easier to catch that way too.

Reply:
I mean I've never seen an import specifier that wasn't an identifier, though I guess it could be an assertion